_data/nav.yml

```diff
@@ -4,11 +4,11 @@
   url: /news/
 - title: Documentation
   url: /doc/
-- title: Google Group
-  url: https://groups.google.com/forum/#!forum/neovim
+- title: Community
+  url: /community/
 - title: Twitter
-  url: http://twitter.com/Neovim
-- title: Bountysource
+  url: https://twitter.com/neovim
+- title: Donations
```

Comment:
Oh no... please not the second one :)

On April 2, 2015 7:35:22 PM EDT, "Justin M. Keyes" <notifications@github.com> wrote:
>>  - title: Twitter
>> -  url: http://twitter.com/Neovim
>> -- title: Bountysource
>> +  url: https://twitter.com/neovim
>> +- title: Donations
>
>I hope we never have the word "donate" anywhere in our materials. It's
>really tacky. If people want to donate they will. How about "Support".
>Or "$upport" if you feel peachy :)
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/neovim/neovim.github.io/pull/91/files#r27706783